pyweek/challenge/templates/base.html

```diff
@@ -110,7 +108,7 @@
             You have {{ unverified_emails }} unverified e-mail address{{ unverified_emails|pluralize:"es" }}.
             Visit <a href="/profile/">your account settings</a> to update and re-send verification.
 	</div>
-	{% elif latest and latest.summary and latest.isRegoOpen %}
+	{% elif latest and latest.summary and not latest.isCompComing and not latest.isAllDone %}
```

Comment:
I think this will not show the final summary message, "<challenge> is finished". That's a useful message only because you can click on it to see the winners and final ratings.

I'm happy to leave the flash until the next competition is added - so just `not latest.isCompComing`, and drop `not latest.isAllDone`.

Reply:
Okay, makes sense.  Fixed.